zmon_aws_agent/aws.py

```diff
@@ -507,6 +507,7 @@ def get_auto_scaling_groups(region, acc, **kwargs):
             'desired_capacity': g['DesiredCapacity'],
             'max_size': g['MaxSize'],
             'min_size': g['MinSize'],
+            'created_time': str(g['CreatedTime']),
```

Comment:
I'm not sure if `str` is redundant here, because this will be later converted to JSON.